ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/TableSelect.java

```diff
@@ -34,14 +34,14 @@
   private final ExecutionStep<KTableHolder<K>> source;
   private final ImmutableList<ColumnName> keyColumnNames;
   private final ImmutableList<SelectExpression> selectExpressions;
-  private final Optional<Formats> internalFormats;
+  private final Formats internalFormats;
```

Comment:
Side cleanup: we always have a format set; no need to make it `Optional`

Reply:
This change needs to be backed out. This is not a backwards compatible evolution. There could be old versions written into the command topic.